### PR TITLE
Regenerate from Elasticsearch master

### PR DESCRIPTION
**Pull request by tylersmalley**
Regenerates master API definition using `generate` npm script.

### CODE REVIEW

File: src/lib/apis/master.js [hunk 1]
```diff
@@ -2708,7 +2722,7 @@ api.indices.prototype.create = ca({
       }
     }
   },
-  method: 'POST'
+  method: 'PUT'
```
Comment (tylersmalley) [inline src/lib/apis/master.js]: Here is the what resolves the index creation issue

**Comment by tylersmalley** [comment]
Here is a reference to the breaking change: https://github.com/elastic/elasticsearch/pull/20001

**Comment by tylersmalley** [comment]
jenkins, test this

**Comment by spalger** [comment]
LGTM

**Pull request merged**